maat/tests/real_world.rs

```diff
@@ -251,7 +251,7 @@ where
         assert_eq!(event.sectors.0, sectors_pre_commit_info);
     }
 
-    result.height
+    client.height(true).await.unwrap()
```

Comment:
It won't work in every case. If we query for it 1ms later and get the next block, instead of the current one, the whole machine will fall down. Lots of stuff depends on this pre-commit height (proofs-related). 
We can let it be, but leave a todo marking that this won't always return the correct height. I will fix it in my PR with making maat work again (MMWA).